migration/src/test/java/org/axonframework/migration/AxonJavaxTest.java

```diff
@@ -0,0 +1,42 @@
+package org.axonframework.migration;
+
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+import static org.openrewrite.maven.Assertions.pomXml;
+
+public class AxonJavaxTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec
+                .parser(JavaParser.fromJavaVersion()
+                        .logCompilationWarningsAndErrors(true)
+                        .classpath("axon-messaging"))
+                .recipe("/META-INF/rewrite/axon-javax.yml", "org.axonframework.axon.UpgradeAxonFramework_4_7_Javax");
+    }
+
+    @Test
+    void migrateImports() {
+        //language=java
```

Comment:
Although extremely verbose, shouldn't this validate *all* changes in the YAML?

Reply:
Left as an exercise to the reader. It's late. ;)